File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/LessThanEqualsOperation.java
package org.evomaster.client.java.controller.mongo.operations;

/**
 * Represent $lte operation.
 * Selects the documents where the value of the field is less than or equal to a specified value
 */
public class LessThanEqualsOperation<V> extends ComparisonOperation<V>{
    public LessThanEqualsOperation(String fieldName, V value) {
        super(fieldName, value);
    }
}

File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/LessThanEqualsOperation.kt
package org.evomaster.client.java.controller.mongo.operations

/**
 * Represent $lte operation.
 * Selects the documents where the value of the field is less than or equal to a specified value
 */
class LessThanEqualsOperation<V>(
    override val fieldName: String,
    override val value: V
): ComparisonOperation<V>(fieldName, value)